### DISCUSSION TITLE
Add category/tag on import step

### DISCUSSION THREAD

**Discussion by hilmarp** [mealie-recipes/mealie#1563] [Feature Request]
**Before submitting this feature request I have**
- [x] Reviewed the documentation and verified that my feature request isn't already a feature.
- [x] Reviewed the existing feature requests and my feature has not been requested.
- [x] Checked the [tasks taged](https://github.com/hay-kot/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

**Please Describe The Problem To Be Solved**

I add a category and tag to all recipes I import, and I know which category/tag the recipe I'm importing should go in before I click "Create".

It would be really nice to be able to do this in one step, at the moment I have to import the recipe, click the "Edit" button, add category/tags, click "Save", it's a lot of clicks.

So in short, adding a category/tag dropdown on the import step (in the first screenshot).

![Screenshot 2022-08-15 at 11 13 01](https://user-images.githubusercontent.com/2643080/184618117-49859e1e-18e7-44a0-988e-7a30b3f85427.png)

**(Optional): Suggest A Solution**

Adding a category/tag dropdown on the import step, similar to the dropdowns on the recipe edit page seen here:

![Screenshot 2022-08-15 at 11 20 58](https://user-images.githubusercontent.com/2643080/184618901-22a8aa8d-6c39-4a95-b450-8f6d43eff215.png)

Then send the selected category/tag to the server along with the URL to scrape, and if the recipe is successfully created then add those categories/tags to the recipe.

**Additional Information**
- [x] If the feature is accepted I'm willing to submit a PR to provide this feature

**Comment by chuckberrypi** [comment]
Chiming in to say this would be a great quality-of-life addition
